concordium-contracts-common/src/schema_json.rs

```diff
@@ -886,10 +1272,114 @@ mod tests {
         let mut cursor = Cursor::new([128, 128, 128, 128, 128, 128, 128, 128, 128, 0b0111_1111]);
         deserial_bigint(&mut cursor, 9).expect_err("Deserialising should fail");
     }
+
+    #[test]
+    fn test_deserial_account_address() {
```

Comment:
Also here. It might be redundant on this particular test case, but it will make the next person adding tests more likely to feel pressure to document his or her test case ;)